pkgs/development/tools/misc/ccache/default.nix

```diff
@@ -15,43 +15,46 @@ let ccache = stdenv.mkDerivation rec {
     substituteInPlace Makefile.in --replace 'objs) $(extra_libs)' 'objs)'
   '';
 
-  doCheck = true;
+  doCheck = !stdenv.isDarwin;
```

Comment:
It's probably not worth trying to fix this if it seems to work. It's always nice to add a comment explaining why tests are disabled if you know why they don't work.